sklearn/ensemble/bagging.py

```diff
@@ -52,6 +53,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
     bootstrap_features = ensemble.bootstrap_features
     support_sample_weight = has_fit_parameter(ensemble.base_estimator_,
                                               "sample_weight")
+    # Logistic regression does not support sample weights with liblinear
+    if (isinstance(ensemble.base_estimator_,LogisticRegression) and 
```

Comment:
Well the model *does* support sample weights, just not with the default solver (`liblinear`), so I'm not sure what the best way to proceed would be. 

Unfortunately raising the error results in a test failure.